docs/eventing/experimental-features.md

```diff
@@ -9,6 +9,10 @@ This document explains how to enable experimental features and which ones are av
 
 For more details about the process, the feature phases, quality requirements and guarantees, check out the [Experimental features process documentation](https://github.com/knative/eventing/blob/main/docs/experimental-features.md).
 
+!!! note
```

Comment:
```suggestion
!!! warning
```

I'd make this a warning instead of a note maybe?

Reply:
Didn't knew this was an option, lemme change that :)